app/models/account.rb

```diff
@@ -2,5 +2,5 @@
 
 class Account < ApplicationRecord
   belongs_to :status
-  validates :UIN, :FirstName, :LastName, :PhoneNumber, :Email, :status_id, presence: true
+  validates :UIN, :FirstName, :LastName, :PhoneNumber, :Email, :status_id, :uuid, presence: true
```

Comment:
uuid is a new attribute which is mandatory so, it may crash heroku since its database doesnt have uuid. I have no idea how to drop database from heroku server's database (drop command doesnt work, atleast for me).

I am going to merge it to main to see if it work, if it doesnt, please help with this.